### PR TITLE
Implemented additional health checks

### PR DESCRIPTION
**Pull request by blootsvoets**
Added health checks for zookeeper, kafka, rest proxy, and the mongodb and hdfs connectors.

### CODE REVIEW

**Comment by yatharthranjan** [comment]
These will be very useful. I have a question. Is there any way to send a notification (like an email) when a container is not healthy? Because it will be helpful to know immediately when something fails.

File: dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml [hunk 1]
```diff
@@ -394,6 +394,11 @@ services:
       CONNECT_CONSUMER_SESSION_TIMEOUT_MS: 10000
       CONNECT_CONSUMER_HEARTBEAT_INTERVAL_MS: 3000
       TOPIC_LIST: ${RADAR_TOPIC_LIST}
+    healthcheck:
+      test: ["CMD-SHELL", "[ \"$$(curl  -sf localhost:8083/connectors/radar-connector-mongodb-sink/status | grep -o '\"state\":\"[^\"]*\"' | sed 's/\".*\":\"\\(.*\\)\"/\\1/' | tr '\\n' ',')\" = 'RUNNING,RUNNING,RUNNING,RUNNING,RUNNING,' ] || exit 1"]
```
Comment (yatharthranjan) [inline dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml]: Always says unhealthy even when the connector is working fine.

**Comment by yatharthranjan** [comment]
@blootsvoets can we also add heathchecks for MP app, postgres and kafka-manager.

**Comment by blootsvoets** [comment]
I'll check the mongodb connector again. I'll take a look at postgres, MP and kafka as well. There are two ways to respond to unhealthy containers:
1. via `docker events`, in that case a daemon needs to be continuously listening
2. via `docker inspect`, in that case you can poll the status whenever you like.

I'm implementing a script for the second option to restart unhealthy services.

**Comment by yatharthranjan** [comment]
This sounds great. I will also look into Puppet for our stack and integrating it. But this should be more than enough for now. Thanks

**Comment by blootsvoets** [comment]
@yatharthranjan I've added the `check-health.sh` script and added health checks for the remaining services.

**Comment by yatharthranjan** [comment]
@blootsvoets Okay great. I will check in a moment. And do we need to start this script in the background and leave it running for monitoring?

**Comment by blootsvoets** [comment]
It should be run as a cron job

**Comment by afolarin** [comment]
closes #69

**Pull request merged**